phpunit/class-gutenberg-rest-global-styles-controller-test.php

```diff
@@ -33,7 +33,6 @@ public function set_up() {
 	 * @param WP_UnitTest_Factory $factory Helper that lets us create fake data.
 	 */
 	public static function wpSetupBeforeClass( $factory ) {
-		gutenberg_register_wp_theme_taxonomy();
```

Comment:
This function is now not defined for Gutenberg running against WP 5.9, as the taxonomy is registered in core. The call here was making the test fail though. I deleted it and the tests still pass, so I'll call that a win 🤷